apps/studio/src/components/AuthWrappers/PermissionsBoundary.tsx

```diff
@@ -20,6 +20,12 @@ const ERROR_COMPONENT_PROPS: Record<ResourceType, ErrorProps> = {
       "To have access, ask your site admins to assign this collection to you",
     buttonText: "Back to My Sites",
   },
+  CollectionMeta: {
```

Comment:
Curious, this meta is not user accessible right? these error messages are user-facing and will only likely apply for things like RootPage, CollectionPage, Folder

Reply:
yeap, it's not - this is required because of the typing